BEST_PRACTICES.md

```diff
@@ -237,40 +239,8 @@ COPY --from=build /hello /hello
 CMD ["/hello"]
 ```
 
-
 Other tests that should be considered:
 
-
-
 * Tests for any configuration options provided e.g. setting a password
 * Mounting data or configuration e.g. running nginx with a HTML directory
 * Connecting via another container
```

Comment:
```suggestion
* Connecting via another container
```

## Sending a Pull Request
We have a number of automated checks that must pass before a pull request is merged. 

Commits must be signed with either [Gitsign](https://docs.sigstore.dev/signing/gitsign/) or with regular [gpg signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). For new images, you'll also need to run the included `monopod` tool to "fix" the image README file. This will lint the file and add a default header with relevant links.

To install monopod:

```shell
cd monopod
go install
```

Running monopod for README checks - this must be executed from the root of the project:

```shell
monopod readme 
```